courses/templates/user/user_courses.html

```diff
@@ -31,7 +31,12 @@ <h3>{% trans "Teaching" %}</h3>
         </p>
     {% endif %}
 
-    <h3>{% trans "My Subscriptions" %}</h3>
+    <div class="row">
+        <h3 class="col-sm-11">{% trans "My Subscriptions" %}</h3>
+        <a href="{% url "user_courses_ical" user.id %}?token={{ token }}" class="col-sm-1" title="{% trans "iCalendar export" %}">        
```

Comment:
```suggestion
        <a href="{% url "user_courses_ical" user.id %}?token={{ token }}" class="col-sm-1" title="{% trans "Add to calendar" %}">        
```

Maybe changing the label of the link? I think iCalendar export sounds a bit too technical for the majority of users